### PR TITLE
Add GitHub Actions CI

### PR DESCRIPTION
**Pull request by eumiro**
Since the `master` still does not use this, the results won't be immediately visible in this PR, so you may have to test it in an extra branch on the main repo and only when it works (or needs bugfixing), you can merge it into `master`.

### CODE REVIEW

**Comment by eumiro** [comment]
It looks like the [run](https://github.com/eumiro/py3status/actions/runs/418559544) was successful.

**Comment by ultrabug** [comment]
Very interesting thanks @eumiro 

I guess we could drop travis in that case right?

File: tox.ini [hunk 1]
```diff
@@ -34,3 +35,11 @@ flake8-ignore =
 flake8-select = F
 flake8-max-complexity = 18
 flake8-max-line-length = 88
+
+[gh-actions]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38, mypy
```
Comment (ultrabug) [inline tox.ini:43]: can you explain mypy wrt 3.8 only plz?

Comment (eumiro) [reply tox.ini:43]: This comes from the [`tox-gh-actions`](https://github.com/ymyzk/tox-gh-actions) page. I believe it's enough to put it along a single version.

**Comment by eumiro** [comment]
> Very interesting thanks @eumiro
> 
> I guess we could drop travis in that case right?

Yes, but just let run both next to each other for a few more PRs before you drop travis.

**Comment by ultrabug** [comment]
Thanks!

**Pull request merged**